README.md

```diff
@@ -1,9 +1,18 @@
 # Carpoolear frontend
 
+Versión en Español:
+
 Carpoolear es la primera aplicación argentina de Facebook que permite a los usuarios de dicha red social compartir viajes en automóvil con otros usuarios de su entorno.
 
 Es una customización ad-hoc para Argentina de la filosofía carpooling, la cual consiste en compartir nuestros viajes en auto con otras personas de forma cotidiana. El carpooling es una práctica popular en Estados Unidos y Europa, donde se realiza de manera organizada para lograr aumentar el número de viajes compartidos y que estos sean concretados con otras personas además de nuestros vecinos y amigos.
 
+English Version:
+
+Carpoolear is the first argentinean app of Facebook that allow users to share travels in car with others users.
+
+Is a customization ad-hoc for Argentina of the carpooling philosophy, the wich one consist on share our travels in car with other people on a regular basis. The carpooling is a popular practicar in USA and Europe, where it is done in an organized way to increase the number of shared travels and let that be done with other people besides our neighbors and friends.
```

Comment:
Is an ad-hoc customization of the carpooling philosophy  for Argentina, that consist on share car trips